docs/source/examples.rst

```diff
@@ -0,0 +1,226 @@
+    #  array([18.87013369, -3.11936304]),
+    #  array(1.44847934),
+    #  array([ 9149554, 13446053])]
+
+    # Compile the joint log-density function
+    logprob_fn = aesara.function([a, b, X] + list(value_variables), logprob)
+    print(logprob_fn(1., 1., np.ones((2, 2)), *sample))
+    # -50.34214668084496
+
+
+Discrete HMM
+============
+
+AePPL allows to condition on random variables that are generated inside a loop, which allows one to express discrete Hidden Markov Models naturally:
+
+.. code::
+
+    import aeppl
+    import aesara
+    import aesara.tensor as at
+
+    srng = at.random.RandomStream()
+
+    N_tt = at.iscalar("N")
+    M_tt = at.iscalar("M")
+    mus_tt = at.matrix("mus_t")
+
+    sigmas_tt = at.ones((N_tt,))
+    Gamma_rv = srng.dirichlet(at.ones((M_tt, M_tt)), name="Gamma")
+
+    def scan_fn(mus_t, sigma_t, Gamma_t):
+        S_t = srng.categorical(Gamma_t[0], name="S_t")
+        Y_t = srng.normal(mus_t[S_t], sigma_t, name="Y_t")
+        return Y_t, S_t
+
+    (Y_rv, S_rv), _ = aesara.scan(
+        fn=scan_fn,
+        sequences=[mus_tt, sigmas_tt],
+        non_sequences=[Gamma_rv],
+        outputs_info=[{}, {}],
+        strict=True,
+        name="scan_rv",
+    )
+
+    logprob, value_variables = aeppl.joint_logprob(Gamma_rv, Y_rv, S_rv)
+
+
+The PERT distribution
+=====================
+
+AePPL supports many :doc:`random variables <api/distributions>` out of the box; if you cannot find the distribution you need, however, there's a good chance you can express it as the transformation of other random variables.
+
+The `PERT distribution <https://en.wikipedia.org/wiki/PERT_distribution>`_, for instance, is a tranformation of the Beta distribution. AePPL allows you to express PERT-distributed random variable as a transformation and condition on this random variable in a log-density graph:
+
+.. testcode::
+
+    import aeppl
+    import aesara
+    import aesara.tensor as at
+
+    srng = at.random.RandomStream(0)
+
+    def pert(srng, a, b, c):
+        r"""Construct a random variable that is PERT-distributed."""
+        alpha = 1 + 4 * (b - a) / (c - a)
+        beta = 1 + 4 * (c - b) / (c - a)
+
+        X_rv = srng.beta(alpha, beta)
+
+        z = a + (b - a) * X_rv
+
+        return z
+
+    A_rv = srng.uniform(10, 20, name="A")
+    B_rv = srng.uniform(20, 65, name="B")
+    C_rv = srng.uniform(65, 100, name="C")
+    Y_rv = pert(srng, A_rv, B_rv, C_rv)
+
+    logprob, (y_vv, a_vv, b_vv, c_vv) = aeppl.joint_logprob(Y_rv, A_rv, B_rv, C_rv)
+
+    # Compile a function that samples from the prior predictive distribution
+    sample_fn = aesara.function([], [Y_rv, A_rv, B_rv, C_rv])
+    sample = sample_fn()
+    print(sample)
+    # [array(25.51948424), array(19.42937553), array(50.47385856), array(94.33949018)]
+
+    # Compile the joint log-density function
+    logprob_fn = aesara.function([y_vv, a_vv, b_vv, c_vv], logprob)
+    print(logprob_fn(*sample))
+    # -12.956702290497232
```

Comment:
Looks like these `testcode` blocks aren't showing up: [here](https://aeppl--213.org.readthedocs.build/en/213/examples.html#the-pert-distribution).